### ISSUE TITLE
No compatible version found: babel-runtime@^6.22.0

### ISSUE THREAD

**Issue by mattmull** [babel/babel#5160]
I believe the same issue https://github.com/babel/babel/issues/4975 that happened when 6.20 was released is happening again. 

### Babel Configuration (.babelrc, package.json, cli command)
I've cleared cache and checked for the version again and still get 6.20.0
```
>npm cache clear
>npm view babel-runtime version
6.20.0
```

### Expected Behavior
Running `npm i` completes successfully.

### Current Behavior
`npm i` causes:
```
npm ERR! No compatible version found: babel-runtime@^6.22.0
npm ERR! Valid install targets:
npm ERR! 6.20.0, 6.18.0, 6.11.6, 6.9.2, 6.9.1, 6.9.0, 6.6.1, 6.6.0, 6.5.0, 6.5.0-1, 6.3.19, 6.3.13, 6.2.4, 6.2.0, 6.1.18, 6.1.17, 6.1.4, 6.0.14, 6.0.12, 6.0.8, 6.0.2, 5.8.38, 5.8.35, 5.8.34, 5.8.29, 5.8.25, 5.8.24, 5.8.20, 5.8.19, 5.8.12, 5.8.9, 5.8.8, 5.8.5, 5.8.3, 5.8.2, 5.7.0, 5.6.20, 5.6.19, 5.6.18, 5.6.17, 5.6.16, 5.6.15, 5.6.14, 5.6.12, 5.6.11, 5.6.10, 5.6.9, 5.6.8, 5.6.7, 5.6.6, 5.6.5, 5.6.4, 5.6.3, 5.6.2, 5.6.1, 5.6.0, 5.5.8, 5.5.7, 5.5.6, 5.5.5, 5.5.4, 5.5.3, 5.5.2, 5.5.1, 5.5.0, 5.4.7, 5.4.6, 5.4.5, 5.4.4, 5.4.3, 5.4.2, 5.4.1, 5.4.0, 5.3.3, 5.3.2, 5.3.1, 5.3.0, 5.2.17, 5.2.16, 5.2.15, 5.2.14, 5.2.13, 5.2.12, 5.2.11, 5.2.10, 5.2.9, 5.2.7, 5.2.6, 5.2.5, 5.2.4, 5.2.3, 5.2.2, 5.2.1, 5.2.0, 5.1.13, 5.1.12, 5.1.11, 5.1.10, 5.1.9, 5.1.8, 5.1.7, 5.1.6, 5.1.5, 5.1.4, 5.1.3, 5.1.2, 5.1.1, 5.1.0, 5.0.13, 5.0.12, 5.0.11, 5.0.10, 5.0.9, 5.0.8, 5.0.7, 5.0.6, 5.0.5, 5.0.4, 5.0.3, 5.0.2, 5.0.1, 5.0.0, 5.0.0-beta4, 5.0.0-beta3, 5.0.0-beta2, 5.0.0-beta1, 4.7.16, 4.7.15, 4.7.14, 4.7.13, 4.7.12, 4.7.11, 4.7.10, 4.7.9, 4.7.8, 4.7.7, 4.7.6, 4.7.5, 4.7.4, 4.7.3, 4.7.2, 4.7.1, 4.7.0, 4.6.6, 4.6.5, 4.6.4, 4.6.3, 4.6.1, 4.6.0, 4.5.5, 4.5.4, 4.5.3, 4.5.2, 4.5.1, 4.5.0, 4.4.6, 4.4.5, 4.4.4, 4.4.3, 4.4.2, 4.4.1, 4.3.0, 4.2.1, 4.2.0, 4.1.1, 4.0.2, 4.0.1
```

### Possible Solution
Seems last time, npmjs cache hadn't been refreshed.

**Comment by hzoo** [comment]
Man that really sucks.

You're using the regular registry right?

`npm view babel-runtime version` >> 6.22.0 for me.

https://unpkg.com/babel-runtime@6.22.0/ is also the latest

Sorry can't help more atm, going to rest

**Comment by mattmull** [comment]
Yes, npmjs also shows 6.20.0 as the latest.
<img width="400" alt="screen shot 2017-01-19 at 11 22 24 pm" src="https://cloud.githubusercontent.com/assets/9367752/22135747/306d87de-de9e-11e6-852c-4fbb5e206e9b.png">

**Comment by existentialism** [comment]
re: npm website version => https://github.com/babel/babel/issues/4975#issuecomment-266171772

**Comment by hzoo** [comment]
It's also 6.22 on https://registry.npmjs.org/babel-runtime/

**Comment by mattmull** [comment]
My mistake - I noticed we are using a virtual npmjs repo (we pull/cache npmjs repo for internal use) and that is where the old version is coming from. If I bypass that and go directly to npmjs it resolves as expected.

**Comment by existentialism** [comment]
@mattmull no worries, glad it's resolved!

**Comment by rkirsling** [comment]
I experienced the same issue in a GitHub Enterprise / NPM Enterprise environment.

Around 5:00 PT today, Greenkeeper starting firing off broken branches like the world was ending:
![](https://cloud.githubusercontent.com/assets/2172590/22136570/18b33260-de8a-11e6-8452-7e936e6ad508.png)
All of the errors were `No compatible version found: babel-...@^6.22.0`.

I was able to experience this behavior locally as well:
`npm info` would display `6.22.0`, but `npm install` would find nothing (even after `npm cache clean`).

When I tried again around 7:00 PT, the issue had simply disappeared.
